### DISCUSSION TITLE
About DataRequest Class

### DISCUSSION THREAD

**Discussion by cb-venkatesh-nellore** [Alamofire/Alamofire#3827] [General]
Hi 
As i can see i'm getting issue while using DataRequest.serializeResponse method i'm not able to find replacement in the document can anyone please help me. with that if possible

**Comment by jshier** [comment]
Do you mean from Alamofire 4? If so, the replacement is to use a proper `ResponseSerializer` and the modern serializer pipeline. But without more context I can't really help you.

**Comment by James09934** [comment]
thank you for sharing this information....
[Marion County Property Appraiser](https://marioncountypropertyappraiser.site)
